src/test/scala/nl/knaw/dans/easy/fedoratobag/DdmSpec.scala

```diff
@@ -465,66 +474,11 @@ class DdmSpec extends TestSupportFixture with EmdSupport with AudienceSupport wi
         </ddm:dcmiMetadata>
       </ddm:DDM>
       )
-    ) // note that a missing x or y defaults to zero
-    assume(schemaIsAvailable)
-    triedDDM.flatMap(validate) shouldBe Success(())
-  }
-
-  it should "should ignore empty place element" in {
```

Comment:
why was this test removed? Is this covered elsewhere?

Reply:
as the commit message says: merged tests